datalad/core/distributed/clone.py

```diff
@@ -653,12 +646,16 @@ def postclonecfg_ria(ds, props):
                 lgr.debug("Failed to get config file from source: %s",
                           exc_str(e))
         else:
-            lgr.debug("Unknown URL-Scheme in %s. Can currently handle SSH or "
+            lgr.debug("Unknown URL-Scheme in %s. Can handle SSH, HTTP or "
                       "FILE scheme URLs.", props['source'])
 
         # 3. And read it
         org_uuid = None
         if config_content:
+            # TODO: We might be able to spare the saving to a file.
+            #       "git config -f -" is not explicitly documented but happens
+            #       to work and would read from stdin. Make sure we know this
+            #       works for required git versions and on all platforms.
```

Comment:
FWIW the stdin behavior is explicitly checked in Git's tests and has been around since v2.0.0, specifically 3caec73b55 (config: teach "git config --file -" to read from the standard input, 2014-02-19).